src/components/editor/ResourceTemplate.jsx

```diff
@@ -50,6 +51,7 @@ class ResourceTemplate extends Component {
         <div id="resourceTemplate" style={{ marginTop: '-30px' }}>
           <section className="col-md-9">
             <h1><em>{this.props.title}</em></h1>
+            <BaseURL />
```

Comment:
if there is a problem saving ... we don't want this to show, correct?  We would want the error message modal and to NOT show new component?

Reply:
It shouldn't show up if there is a problem I believe, but will try to verify.

Reply:
wonder if it would be worth writing a test (with a mock, not with network call to server)

Reply:
I'll try to get a test going, confirmed that it does not show up if there is an error.